common/v2/features/AddAccount/components/LedgerNano.tsx

```diff
@@ -150,6 +146,11 @@ class LedgerNanoSDecryptClass extends PureComponent<Props, State> {
       });
   };
 
+  private getInitialDPath() : DPath {
+    const network = getNetworkByName(this.props.formData.network);
+    return network && network.dPaths.ledgerNanoS ? network.dPaths.ledgerNanoS : this.props.dPaths[0]
+  }
+
```

Comment:
It seems this method is defined 6 times. Please turn into a pure function and place in EthService/network/helpers and import from `v2/services/EthService`